sympl/_core/util.py

```diff
@@ -652,3 +653,42 @@ def get_final_shape(data_array, out_dims, direction_to_names):
                 np.product([len(data_array.coords[name])
                             for name in direction_to_names[direction]]))
     return final_shape
+
+
+def get_component_aliases(components):
+    """
+    Returns aliases for variables in the properties of Components (e.g., Prognostics).
+
+    Args
+    ----
+    components : iterable of Component
+        Components from which to fetch variable aliases from the input_properties,
+        diagnostic_properties, and tendency_properties dictionaries
+
+    Returns
+    -------
+    aliases : dict
+        A dictionary with keys containing old variable names and values containing
+        new variable names
+    """
+
+    aliases = {'grid_latitude': 'latitude', 'grid_longitude': 'longitude'}
+
+    # First handle the input_properties and the diagnostic_properties
+    component_properties = [{**comp.diagnostic_properties, **comp.input_properties}
```

Comment:
Change this to
```
component properties = {}
for comp in components:
    component_properties.update(comp.diagnostic_properties)
    component_properties.update(comp.input_properties)
```
Then change the below code to reflect the fact that you have a single dict, and not a list of dicts.